Dockerfile

```
@@ -1,37 +1,50 @@
+LABEL software.version="0.0.2"
+LABEL about.summary="pyspectrafuse - Command-line utilities for spectrum clustering and conversion"
+LABEL about.home="https://github.com/bigbio/pyspectrafuse"
+LABEL about.documentation="https://github.com/bigbio/pyspectrafuse"
+LABEL about.license_file="https://github.com/bigbio/pyspectrafuse/blob/master/LICENSE"
+LABEL about.license="SPDX:Apache-2.0"
+LABEL about.tags="Proteomics,Multiomics,QuantMS"
 
-# Set working directory
-WORKDIR /app
+################## MAINTAINER ######################
+MAINTAINER Yasset Perez-Riverol <ypriverol@gmail.com>
```

Hadolint (2.14.0)

[error] MAINTAINER is deprecated

(DL4000)